components/atoms/heading/heading.js

```diff
@@ -38,7 +38,13 @@ const StyledHeading = styled.span`
     text-transform: ${theme.HEADINGS[as].textTransform};
     margin: 0;
   `}
-
+  ${({ ellipsis }) =>
```

Comment:
@mhsnprvr3 The Trucncation UTIL should be used here instead of duping the code ?